hiera.yaml

```diff
@@ -6,6 +6,9 @@ defaults:
   data_hash: 'yaml_data'
 
 hierarchy:
+  - name: 'Set important hiera options'
+    path: "hiera_options.yaml"
+
```

Comment:
If we keep the default behavior, no need for this :wink:

Reply:
see above, I think it is necessary to avoid unexpected breakage in some environments